src/components/resource-selector/media-library/media-library-metadata-form.js

```diff
@@ -47,7 +47,7 @@ function MediaLibraryMetadataForm({ form, file, useOptimizeImage, disableOptimiz
           <Info tooltip={t('common:shortDescriptionInfo')} iconAfterContent>{t('common:shortDescription')}</Info>
         }
         >
-        <TextArea rows={3} maxLength={maxMediaLibraryItemShortDescriptionLength} />
+        <NeverScrollingTextArea rows={3} maxLength={maxMediaLibraryItemShortDescriptionLength} />
```

Comment:
Otherwise the max length counter would not be visible, also we use it for other `shortDescription` fields